docs/src/Tooltip.doc.js

```diff
@@ -1,165 +1,860 @@
-      <Button
-        accessibilityLabel="This tooltip wraps an inline button and occupies the same space"
-        color="gray"
-        text="Inline"
+  <MainSection name="Variants">
+    <MainSection.Subsection
+      title="Ideal direction"
+      description={`Pass in \`idealDirection\` to specify the preferred position of Tooltip relative to its anchor element. The default direction is "down", which should be used in most cases. The actual position may change given the available space around the anchor element.`}
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+        function ExampleTooltip() {
+  const [idealDirection, setIdealDirection] = React.useState('down');
+
+  return (
+    <Flex direction="column" alignItems="center" gap={8}>
+      <Flex gap={4}>
+        <RadioButton
+          checked={idealDirection === 'up'}
+          id="up"
+          label="Up"
+          name="idealDirection"
+          onChange={() => setIdealDirection( 'up' )}
+          value="up"
+        />
+        <RadioButton
+          checked={idealDirection === 'right'}
+          id="right"
+          label="Right"
+          name="idealDirection"
+          onChange={() => setIdealDirection( 'right' )}
+          value="right"
+        />
+        <RadioButton
+          checked={idealDirection === 'down'}
+          id="down"
+          label="Down"
+          name="idealDirection"
+          onChange={() => setIdealDirection( 'down' )}
+          value="down"
+        />
+        <RadioButton
+          checked={idealDirection === 'left'}
+          id="left"
+          label="Left"
+          name="idealDirection"
+          onChange={() => setIdealDirection( 'left' )}
+          value="left"
+        />
+      </Flex>
+      <Tooltip
+        idealDirection={idealDirection}
         inline
+        text="Share"
+        >
+          <IconButton
+            accessibilityLabel="Share this Pin"
+            bgColor="white"
+            icon="share"
+            iconColor="darkGray"
+            inline
+            size="lg"
+          />
+        </Tooltip>
+      </Flex>
+
+  )
+}
+`}
       />
-    </Tooltip>
-  </Box>
-  <Box padding={2}>
-    <Tooltip text="Full width tooltip">
-      <Button
-        accessibilityLabel="This tooltip wraps and allows a button to remain full width"
-        color="red"
-        text="Full width"
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Inline"
+      description="Use inline to properly position Tooltip relative to an inline element, such as an [Icon Button](/IconButton)"
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+function SectionsIconButtonDropdownExample() {
+  const [open, setOpen] = React.useState(false);
+  const anchorRef = React.useRef(null);
+
+  return (
+    <Box width={600}>
+      <Heading accessibilityLevel={1}>Sugar-Free Strawberry-Chocolate Greek Yogurt Bark Three-Step Recipe.
```

Comment:
this will need to use the size prop to make it large with accessibilityLevel={4}